### DISCUSSION TITLE
Please share your experiences. How are you using this container?

### DISCUSSION THREAD

**Discussion by ehfd** [selkies-project/docker-selkies-egl-desktop#16] [Show and tell]
This project started out as a research task, and we expect many users to use this container for Cloud Gaming, XR, Autonomous Driving, Robotics, and many other advanced technology development projects.

You will help us greatly if you could share the applications of which this project is being used, however detailed you may explain.
This will also provide new ideas across different fields to the community for incubating interdisciplinary research.
Please also write your laboratory or corporation if possible, and the list of publications or results that used/was influenced by this project.

Please also check this [Powerpoint](https://www.researchgate.net/publication/353658640_GPU_Remote_Desktop_Container_Project) if you want more information about this container.

**Comment by shepherdjerred** [comment]
This is such a great project. Thank you for all of your work on this.

I spent the last weekend looking into ways to stream video to Discord in Docker. The end goal is to have something like Twitch Plays Pokemon, but for Discord. If you're not familiar, the concept is to let a group of people play a shared game of Pokemon by sending controller commands as text, e.g. sending the message "A" would be the same as pressing the A button on a controller. Feedback is given to players through a Discord video stream, which requires hardware acceleration to be performant.

I'm using this container to run Puppeteer which controls Chrome and Discord. Puppeteer also receives input from Discord text chat and translates them into game commands. I worked on many other approaches, and this has by far been the easiest.

https://github.com/shepherdjerred/discord-plays-pokemon
